<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pageprincipal</class>
 <widget class="QMainWindow" name="Pageprincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>577</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Pageprincipal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="gridWidget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1920</width>
        <height>1080</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0" columnstretch="0">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
       </property>
       <item row="4" column="0" rowspan="2">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <widget class="QPushButton" name="spectre_selection">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Tracer le spectre de la selection</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="Exporterselection">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Exporter la selection</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="Suppr_selection">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Supprimer la selection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGraphicsView" name="graphicsView">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777212</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
         </property>
         <item row="1" column="0">
          <widget class="QPushButton" name="BonttonEnregistrer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Enregistrer</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::MediaRecord"/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="Enregistrer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Enregistrer un nouveau Signal</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
           </property>
           <item>
            <widget class="QPushButton" name="BouttonLire">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Lire</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BouttonPause">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Pause</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BoutonArreter">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Arreter</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::MediaPlaybackStop"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="Devices">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Entrez votre périphérique d'entrée</string>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::RichText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QComboBox" name="Devicesout">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Entrez votre périphérique de sortie</string>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::RichText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="Lire">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Lire le signal</string>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::PlainText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionImporter"/>
    <addaction name="actionExporter_csv"/>
    <addaction name="actionExporter_wav"/>
   </widget>
   <widget class="QMenu" name="menuAnalyse">
    <property name="title">
     <string>Analyse</string>
    </property>
    <addaction name="actionTracer_le_spectre"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAnalyse"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionImporter">
   <property name="text">
    <string>Importer</string>
   </property>
  </action>
  <action name="actionTracer_le_spectre">
   <property name="text">
    <string>Tracer le spectre</string>
   </property>
  </action>
  <action name="actionExporter_csv">
   <property name="text">
    <string>Exporter (.csv)</string>
   </property>
  </action>
  <action name="actionExporter_wav">
   <property name="text">
    <string>Exporter(.wav)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
